.github/workflows/npm.yml

```diff
@@ -55,6 +55,9 @@ jobs:
       - name: Disable git safe directories as we are in a container.
         run: git config --global --add safe.directory '*'
 
+      - name: Allow bower to run as root.
+        run: echo '{ "allow_root": true }' > /root/.bowerrc
```

Comment:
Do all of our sites have Bower? Is it worth adding a check for `if [ -f /root/.bowerrc ]; then echo....` or something?

Reply:
No, they don't, but if they don't this is harmless. TBH it might be better to put in the upstream image...